### PR TITLE
Feat(cvp_configlet_upload): Expand prefix options to not modify configlet name

### PR DESCRIPTION
**Pull request by pvinci-arista**
## Related Issue(s)

Fixes #1946 

## Component(s) name

`arista.avd.cvp_configlet_upload`

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by gmuloc — COMMENTED** [review]
Looks good - could you add some tests for `None` and `""` in this file as well: https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/tests/unit/modules/test_configlet_build_config.py#L54

**Review by ClausHolbechArista — COMMENTED** [review]
Please update the PR description and title to reflect the updates you are suggesting. (I agree on your changes).
The PR title is directly used in the release-notes, so it should be precise.

File: ansible_collections/arista/avd/tests/unit/modules/test_configlet_build_config.py [hunk 1]
```diff
@@ -51,8 +51,9 @@ def test_get_configlet_invalid_source(self):
         output = get_configlet()
         assert output == dict()
 
-    def test_get_configlet_none_prefix(self):
+    @pytest.mark.parametrize("prefix", ['none', "", None])
+    def test_get_configlet_none_prefix(self, prefix):
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/tests/unit/modules/test_configlet_build_config.py:55]: There is another test "test_get_configlet_blank_prefix" which fails now. Since you are now including that scenario here, the other test should be removed. But this also means that the changes are actually breaking in case someone has a blank prefix today. 
 @carlbuchmann can we accept this for minor?
